### PR TITLE
Inputs and slider

### PR DESCRIPTION
**Pull request by rahulk31**
Added Input and slider components

### CODE REVIEW

File: css/components/button.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
   border-radius: 5px;
   border: none;
   font-family: inherit;
-  box-shadow: 0 0 20px 15px rgba(0, 0, 0, 0.06);
+  box-shadow: 0 0 15px 10px rgba(0, 0, 0, 0.05);
```
Comment (j24m) [inline css/components/button.css:14]: I think it's suggested to use rems for more than 10px, and use color variables everywhere.

File: css/components/input.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.input-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin: 2rem;
+  padding: 2rem;
+  box-shadow: 0 0 15px 10px var(--box-shadow-black05);
```
Comment (j24m) [inline css/components/input.css:8]: use rems here too

File: css/components/slider.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  align-items: center;
+  justify-content: space-between;
+}
+
+.slider {
+  -webkit-appearance: none;
+  min-width: 90%;
+  height: 0.7rem;
+  background-color: var(--site-secondary);
+  border-radius: 11px;
```
Comment (j24m) [inline css/components/slider.css:15]: rem.

**Review by j24m — CHANGES_REQUESTED** [review]

**Review by varun1233 — APPROVED** [review]
Code looks good, after completing the suggested rem changes its good to merge

**Pull request merged**